src/platform/Kconfig

```diff
@@ -345,7 +345,7 @@ config SYSTICK_PERIOD
 
 config HAVE_AGENT
 	bool "Enable system agent"
-	default y
+	default n
```

Comment:
Lets not disable the agent, but disable the panic. i.e. we would still expect the log to show any critical scheduling delays to aid with further debug.

Reply:
Add a second kconfig that says whether agent timeouts should panic?

Reply:
But why does it panic in the first place? I would expect a panic to mean the DSP cannot operate and needs to be reset. I don't get why removing the panic is a good thing. What exactly does the SA do?

Reply:
@plbossart SA simply notices if the DSP is overloaded and stops it with an immediate panic. I think the intent is that the kernel will respond to the panic by resetting the DSP and reloading the firmware.

Reply:
SA 1.0 would monitor WFI entry and delays i.e. no WFI in a time span would result in panic as we would be spinning somehwere.
SA 2.0 monitors LL scheduling. i.e. if for some reason we cant prempt and LL schedule withing a timout then something is really wrong e.g. IRQ handling spinning,

Reply:
@lgirdwood The problem is that a DSP overload will also trigger an agent panic, and when the load is close to 100% you will get those overloads. And the thing is that with the agent off those overloads just translate into xruns or even just minor glitches in the audio output or input. (YIKES I now remember one issue from a while, the agent would have complained very early on, as in less than a second in the first playback...)

Reply:
EDF scheduling is known to be problematic if the DSP is overloaded, so maybe that was the reason for a panic?
But we have mechanisms to recover from an underflow so if this was a temporary glitch it's certainly less bad than a panic which forces a reboot.